ParquetClassLibrary/Sandbox/Room.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using ParquetClassLibrary.Sandbox.Parquets;
+using ParquetClassLibrary.Stubs;
+
+namespace ParquetClassLibrary.Sandbox
+{
+    /// <summary>
+    /// Models the minimum requirements for a <see cref="Room"/> to be recognizable and useful.
+    /// </summary>
+    public class Room
+    {
+        /// <summary>
+        /// The <see cref="Space"/>s on which a <see cref="Characters.Being"/>
+        /// may walk within this <see cref="Room"/>.
+        /// </summary>
+        public readonly HashSet<Space> WalkableArea;
+
+        /// <summary>
+        /// The <see cref="Space"/>s whose <see cref="Block"/>s and <see cref="Furnishing"/>s
+        /// define the limits of this <see cref="Room"/>.
+        /// </summary>
+        public readonly HashSet<Space> Perimeter;
+
+        /// <summary>
+        /// The cached <see cref="EntityID"/>s for every <see cref="Furnishing"/> found in this <see cref="Room"/>
+        /// together with the number of times that furnishing occurs.
+        /// </summary>
+        private Dictionary<EntityID, int> _cachedFurnishings;
+
+        /// <summary>
+        /// The <see cref="EntityID"/>s for every <see cref="Furnishing"/> found in this <see cref="Room"/>
+        /// together with the number of times that furnishing occurs.
+        /// </summary>
+        public Dictionary<EntityID, int> Furnishings
```

Comment:
future note: This style of caching means that if the Room ever changes this class needs to be discarded/the cache needs to be reset. Watch out for that.

It's a good style, just putting for future thought.

Reply:
Good catch!   I was actually thinking about this when I wrote it but forgot to implement a means to invalidate the cache.